fraud/service_test.go

```diff
@@ -133,7 +132,7 @@ func TestService_ReGossiping(t *testing.T) {
 		},
 		sync.MutexWrap(datastore.NewMapDatastore()),
 		false,
-		string(p2p.Private),
```

Comment:
y

Reply:
Ideologically, because things outside nodebuilder should not depend on nodebuilder (@Wondertan proud of me?)
More practically, because there is an import cycle if you do this. nodebuilder/p2p now depends on share/eds for the edsstore.Blockstore, but eds depends on fraud, so fraud cannot depend on nodebuilder/p2p.